ticketbuyer/buyer.go

```diff
@@ -323,13 +301,23 @@ func (t *TicketPurchaser) Purchase(height int64) (*PurchaseStats, error) {
 	}
 	ps.PriceNext = sDiffEsts.Expected
 
-	// Scale the average price according to the configuration parameters
-	// to find minimum and maximum prices for users that are electing to
-	// attempting to manipulate the stake difficulty.
 	maxPriceAbsAmt, err := dcrutil.NewAmount(t.cfg.MaxPriceAbsolute)
 	if err != nil {
 		return ps, err
 	}
+	// a sensible default, 25% above avgprice
+	negone, err := dcrutil.NewAmount(-1)
+	if maxPriceAbsAmt == negone {
```

Comment:
Instead of checking if it's maxPriceAbsAmt == negone, can you just do if t.cfg.MaxPriceAbsolute < 0 ? then update the comments to explain that anything < 0 is will use the default max price multiplier (1.25 currently).